src/webgl/shaders/filters/dilate.frag

```diff
@@ -0,0 +1,38 @@
+// Increase the bright areas in an image
+
+precision highp float;
+
+varying vec2 vTexCoord;
+
+uniform sampler2D tex0;
+uniform vec2 texelSize;
+
+float luma(vec3 color) {
+  // based on constants 77, 151, 28 from DILATE in filters.js,
+  // even though that's different than the luminance constants used in GRAY
```

Comment:
this is fine, haha. Probably just different implementers

Reply:
@jwong I think that having it match what is expected for the non-webgl filters feels best, glad to see you leave a comment about it here, I think it's helpful context.